system_modules/napvideo/src/videoplayer.cpp

```diff
@@ -173,10 +173,12 @@ namespace nap
 	}
 
 
-	void VideoPlayer::play(double mStartTime)
+	void VideoPlayer::play(double mStartTime, bool clearTheTextures)
```

Comment:
Isn't it potentially dangerous not to clear textures when the texture size is different?

Reply:
There is a check that checks if the texture needs to be resized upon selectVideo which is the only occurance where the textures need to be resized.

Not clearing the textures means that you won't see a black frame when hitting play when the player is already running